client/my-sites/plans/main.jsx

```diff
@@ -67,12 +67,21 @@ var Plans = React.createClass( {
 			url += '/' + selectedSite.slug;
 		}
 
-		return (
-			<a href={ url } className="compare-plans-link" onClick={ this.recordComparePlansClick }>
-				<Gridicon icon="clipboard" size={ 18 } />
-				{ this.translate( 'Compare Plans' ) }
-			</a>
-		);
+		if ( this.props.selectedSite.jetpack ) {
+			return (
+				<a href={ url } className="compare-plans-link" onClick={ this.recordComparePlansClick }>
+					<Gridicon icon="clipboard" size={ 18 } />
+					{ this.translate( 'Compare Options' ) }
```

Comment:
You should just be able to change this line, without duplicating this whole block.